### PR TITLE
[ADF-4631] Start Process Cloud - Add the start form as part of a process

### PR DESCRIPTION
**Pull request by mauriziovitale**
Ability to pre-fill a form with values
Ability to send variables

**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
not able to start a process with a form

**What is the new behaviour?**
Be able to start a process with a form

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by eromano** [comment]
Some unit test are failing

File: demo-shell/src/app.config.json [hunk 1]
```diff
@@ -739,6 +739,9 @@
     "name": "My Default Name",
     "processDefinitionName": "My default process def name"
   },
+  "adf-cloud-start-process": {
+    "name": "My Default Cloud Name"
```
Comment (DenysVuika) [inline demo-shell/src/app.config.json]: Does it support i18n?

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.spec.ts [hunk 2]
```diff
@@ -60,51 +62,104 @@ describe('StartProcessCloudComponent', () => {
         expect(fixture.componentInstance instanceof StartProcessCloudComponent).toBe(true, 'should create StartProcessInstanceComponent');
     });
 
-    describe('first step', () => {
+    describe('start a process without start form', () => {
 
-        describe('without start form', () => {
+        beforeEach(() => {
```
Comment (DenysVuika) [inline lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.spec.ts]: empty block, can be removed

File: demo-shell/src/app.config.json (hunk 1 above)
Comment (mauriziovitale) [reply demo-shell/src/app.config.json]: what do you mean? This is a custom name that you can add by default

**Pull request merged**